ui/src/components/pages/Pipeline/partials/Graph/partials/CustomEdge/index.tsx

```diff
@@ -52,47 +53,43 @@ const CustomEdge: FC<EdgeProps> = ({
   }
   const getMinWM = () => {
     if (data?.edgeWatermark?.watermarks) {
+      return Date.now() - 678;
```

Comment:
Is this line for test? can we remove it?

Reply:
yes, missed cleaning that.